app/Site/RouteServiceProvider.php

```diff
@@ -91,7 +92,7 @@ protected function mapWebRoutes(): void
             Route::get('demo', [ContentController::class, 'demo'])->name('demo');
             Route::get('demo/error/{code}', [ContentController::class, 'errorDemo'])->name('demo.error');
             Route::get('terms', [ContentController::class, 'terms'])->name('terms');
-            // Route::get('downloads', [DownloadController::class, 'index'])->name('download.index');
+             Route::get('downloads', [DownloadController::class, 'index'])->name('download.index');
```

Comment:
```suggestion
             Route::get('download', [DownloadController::class, 'index'])->name('download.index');
```

Everything else is called "download" (singular), including the link itself. Not sure why the path would be any different.

Reply:
index pages are laid out to be plural

Reply:
It seems then that everything else should be changed to reflect this, eg:
_downloads.blade.php_ instead of _download.blade.php_.